src/constants.ts

```diff
@@ -16,26 +16,38 @@ if (!envAvailable) {
   throw new Error("No suitable .env file found.");
 }
 
+const trimTwitterHandle = (handle: string) => {
+  return handle.toLowerCase().trim().replaceAll(" ", "_").replaceAll("@", "");
+};
+
 dotenv.config({ path: envPath });
 
-export const TWITTER_HANDLE = process.env.TWITTER_HANDLE || "";
-export const TWITTER_USERNAME = process.env.TWITTER_USERNAME || "";
-export const TWITTER_PASSWORD = process.env.TWITTER_PASSWORD || "";
-export const MASTODON_INSTANCE = process.env.MASTODON_INSTANCE || "";
-export const MASTODON_ACCESS_TOKEN = process.env.MASTODON_ACCESS_TOKEN || "";
-export const BLUESKY_INSTANCE = process.env.BLUESKY_INSTANCE || "";
-export const BLUESKY_IDENTIFIER = process.env.BLUESKY_IDENTIFIER || "";
-export const BLUESKY_PASSWORD = process.env.BLUESKY_PASSWORD || "";
-export const INSTANCE_ID = process.env.INSTANCE_ID ?? "instance";
-export const STORAGE_DIR = process.env.STORAGE_DIR ?? process.cwd();
-export const CACHE_PATH = `${STORAGE_DIR}/cache.${INSTANCE_ID.toLowerCase()
+export const TWITTER_HANDLE = trimTwitterHandle(
+  process.env.TWITTER_HANDLE || "",
+);
+export const TWITTER_USERNAME = trimTwitterHandle(
+  process.env.TWITTER_USERNAME || "",
+);
+export const TWITTER_PASSWORD = (process.env.TWITTER_PASSWORD || "").trim();
+export const MASTODON_INSTANCE = (process.env.MASTODON_INSTANCE || "").trim();
+export const MASTODON_ACCESS_TOKEN = (
+  process.env.MASTODON_ACCESS_TOKEN || ""
+).trim();
+export const BLUESKY_INSTANCE = (process.env.BLUESKY_INSTANCE || "").trim();
+export const BLUESKY_IDENTIFIER = (process.env.BLUESKY_IDENTIFIER || "").trim();
+export const BLUESKY_PASSWORD = (process.env.BLUESKY_PASSWORD || "").trim();
```

Comment:
```suggestion
export const BLUESKY_INSTANCE = process.env.BLUESKY_INSTANCE?.trim() || "";
export const BLUESKY_IDENTIFIER = process.env.BLUESKY_IDENTIFIER?.trim() "";
export const BLUESKY_PASSWORD = process.env.BLUESKY_PASSWORD?.trim() || "";
```

Reply:
To keep consistency between all consts, I prefer defining the value and _then_ trim it. But sure, it's not necessary here